src/chrome/content/rules/arte.xml

```diff
@@ -64,19 +57,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
 
 	wrong content:
 		- forums.arteradio.com
+
+	Redirect to HTTP:
+		- info.arte.tv
+
+	Too many redirects:
```

Comment:
@I-have-50-open-PRs-to-HTTPS-EVERYWHERE Do you mean it sometimes redirects to HTTP, but sometimes it doesn't?